<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectManager</class>
 <widget class="QWidget" name="ProjectManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1625</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMessageBox QPushButton  {
    border: none;
    background-color: #212121;
    padding: 4px 16px;
}

QMessageBox QPushButton:hover {
    background-color: #555;
}

#searchIcon {
	background: #303030;
	padding: 4px;
}

/* ========================== */

#lineEdit {
	font-size: 14px;
	background: #2f2f2f;
	padding: 3px;
	border: 0;
}

#newProject {
	background: #4898ff;
	color: white;
}

#newProject:hover {
	background-color: #51a1d6;
}

#samples, #projects {
	padding: 8px;
	color: white;
}

#listWidget, #listWidget_2 {
	background: #fff;
}

#listWidget {
	padding-top: 4px;
	padding-left: 2px;
}

QPushButton, QToolButton {
	border: 0;
	padding: 8px 12px;
	background: #444;
	border-radius: 1px;
}

QPushButton::hover {
	background: #555;
}

QToolButton::hover {
	background: #555;
}

QComboBox {
    background: #1A1A1A;
    border: 0;
}

QComboBox:editable {

}

QComboBox QAbstractItemView::item {
    show-decoration-selected: 1;
}

QComboBox QAbstractItemView::item {
    padding: 6px;
}

QListView::item:selected {
    background: #404040;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: #1A1A1A;
}

/* QComboBox gets the on state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: #1A1A1A;
}

QComboBox QAbstractItemView {
    background-color: #1A1A1A;
    selection-background-color: #404040;
    border: 0;
    outline: none;
}

QComboBox QAbstractItemView::item {
    border: none;
    padding-left: 5px;
}

QComboBox QAbstractItemView::item:selected {
    background: #404040;
    padding-left: 5px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 18px;
    border-left-width: 1px;
}

QComboBox::down-arrow {
    image: url(:/icons/down_arrow_check.png);
	width: 18px;
	height: 14px;
}

QComboBox::down-arrow:!enabled {
    image: url(:/icons/down_arrow_check_disabled.png);
    width: 18px;
    height: 14px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="miscPadding" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="pmContainer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../app/images.qrc">:/images/empty_canvas.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="desktopControls" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="importWorld">
        <property name="text">
         <string>Import Scene</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downloadWorlds">
        <property name="text">
         <string>Download Scenes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="tilePreviewLabel">
        <property name="text">
         <string>Tile Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tilePreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentText">
         <string>Normal</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Huge</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="searchLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="searchIcon">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../app/icons.qrc">:/icons/search.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Search recent scenes...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="browseProjects">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>+ Sample Scenes</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newProject">
        <property name="text">
         <string>+ New Scene</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app/icons.qrc"/>
  <include location="../../app/images.qrc"/>
 </resources>
 <connections/>
</ui>
